src/main/java/com/team701/buddymatcher/services/users/impl/UserServiceImpl.java

```diff
@@ -79,6 +79,7 @@ public void deleteBuddy(Long currentUserId, Long buddyUserId)  throws NoSuchElem
     /**
      * OrderBuddyId in terms of userId_i < userId_j based on database structure.
      * This is to ensure we only have one relation per buddy pair.
+     *
```

Comment:
```suggestion
```
Is it necessary to add an empty line to the doc?

Reply:
This was done through IntelliJ's auto code formatting. Do you want me to remove the newline?

Reply:
> This was done through IntelliJ's auto code formatting. Do you want me to remove the newline?

It would be better without it as it seems inconsistent with the rest of the doc.

Reply:
Yup. I will make the necessary changes then.